Mathlib/Topology/Sheaves/Functors.lean

```diff
@@ -86,15 +86,16 @@ The pullback functor.
 def pullback (f : X ⟶ Y) : Y.Sheaf A ⥤ X.Sheaf A :=
   (Opens.map f).sheafPullback _ _ _
 
-lemma pullback_eq (f : X ⟶ Y) :
-    pullback A f = forget A Y ⋙ Presheaf.pullback A f ⋙ presheafToSheaf _ _ := rfl
+--lemma pullback_eq (f : X ⟶ Y) :
+--    pullback A f = forget A Y ⋙ Presheaf.pullback A f ⋙ presheafToSheaf _ _ := rfl
```

Comment:
Did you intend to delete this?

Reply:
Yes, indeed, as it is no longer true, but the iso version is still there.